py/misc/io.py

```diff
@@ -3,14 +3,25 @@
 import time
 import csv
 
-def write_data(item_data):
+class Output:
+    def __init__(self, good_name, price, old_price, id, category, category_label, date = str(datetime.date.today())):
```

Comment:
Not what I meant! This will still return an error if the input data has a new collumn (say, "brand").